docs/spec_draft.md

```diff
@@ -1450,6 +1451,88 @@ More formally, `result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.sort
+
+### Semantics
+
+Sorts a variadic number of tensors in `inputs` together, according to a custom 
+`comparator`, along the given `dimension` and produces a variadic number of
+tensors as `results`. If `is_stable` is true, then the sorting is stable, that
+is, relative order of elements considered to be equal by the comparator is
+preserved.
+
+More formally, for all `0 <= id < jd < di`, either 
```

Comment:
Given that `di` is not defined anywhere, let's use the unambiguous form: `dim(inputs[0], d)`.